src/components/usage-page/content.tsx

```diff
@@ -156,6 +156,49 @@ export default function (version: string, name = "heart", suffix = "") {
 </ion-button>`
       )}
 
+      <h2>Browser support</h2>
+      <p>Ionicons are supported on the browsers listed below:</p>
+      <table>
+        <thead>
```

Comment:
I wonder if the table should have a [caption for web accessibility](https://developer.mozilla.org/en-US/docs/Learn/HTML/Tables/Advanced#adding_a_caption_to_your_table_with_caption) or if the paragraph above would be enough. Thoughts?

Reply:
Oof, I don't think I'd ever tried to navigate a table with a screen reader before. Before I tried it, I was thinking the table was too small to warrant one, since the caption won't be much different than this paragraph. After I tried it, I added the caption. 😅

Reply:
Tables are one of the elements that can easily become non-compliant. I've experienced it too many times that I'm always suspicious of them 😅 😭